frontend/app/components/ItemCategory.tsx

```diff
@@ -0,0 +1,132 @@
+'use client';
+import React, { useState, useEffect } from "react";
+import Image from "next/image";
+
+export default function ItemCategory() {
+
+    // --------------------------------------------------------
+
+    const testCategoryItems = [
+        {
+            toolID: 1,
+            name: "Hunting Axe",
+            lender: "Jarl Balgruff",
+            lenderRating: "4.9",
+            location: "Calgary, AB",
+            image: "/axe.jpg"
+        },
+        {
+            toolID: 2,
+            name: "Woodcutting Axe",
+            lender: "Jarl Balgruff",
+            lenderRating: "4.9",
+            location: "Calgary, AB",
+            image: "/axe.jpg"
+        },
+        {
+            toolID: 3,
+            name: "Hatchet",
+            lender: "Jarl Balgruff",
+            lenderRating: "4.9",
+            location: "Calgary, AB",
+            image: "/axe.jpg"
+        },
+        {
+            toolID: 4,
+            name: "Fire Axe",
+            lender: "Jarl Balgruff",
+            lenderRating: "4.9",
+            location: "Calgary, AB",
+            image: "/axe.jpg"
+        },
+        {
+            toolID: 5,
+            name: "Carpenter Axe",
+            lender: "Jarl Balgruff",
+            lenderRating: "4.9",
+            location: "Calgary, AB",
+            image: "/axe.jpg"
+        },
+        {
+            toolID: 6,
+            name: "Tomahawk Axe",
+            lender: "Jarl Balgruff",
+            lenderRating: "4.9",
+            location: "Calgary, AB",
+            image: "/axe.jpg"
+        },
+        {
+            toolID: 7,
+            name: "Felling Axe",
+            lender: "Jarl Balgruff",
+            lenderRating: "4.9",
+            location: "Calgary, AB",
+            image: "/axe.jpg"
+        },
+        {
+            toolID: 8,
+            name: "Hunting Axe",
+            lender: "Jarl Balgruff",
+            lenderRating: "4.9",
+            location: "Calgary, AB",
+            image: "/axe.jpg"
+        }
+    ]
+
+    const testCategoryName = "Axes"
+
+    // --------------------------------------------------------
+
+    interface ToolItem {
+        toolID: any;
+        name: any;
+        lender: any;
+        lenderRating: any;
+        location: any;
+        image: any;
+    }
+
+    const [categoryName, setCategoryName] = useState("")
+
+    const [categoryItems, setCategoryItems] = useState<ToolItem[]>([])
+
+    const getItemsInCategory = () => {
+        setCategoryName(testCategoryName)
+        // query database to retrieve the items from the selected category
+        // for now, test data was created in place
+        setCategoryItems(testCategoryItems)
+    }
+
+    const viewToolDetails = (toolID: any) => {
+        // will navigate to the tool details page for whichever tool is clicked on
+        // use react router
+    }
+
+    useEffect(() => {
+        getItemsInCategory()
+    })
```

Comment:
A lot of this looks unimplmeneted. We need everything to be working by Wednesday, March 20.